test_cases/autocomplete_street_centroids.json

```diff
@@ -0,0 +1,146 @@
+{
+  "name": "autocomplete street centroids",
+  "priorityThresh": 1,
+  "endpoint": "autocomplete",
+  "tests": [
+    {
+      "id": 1,
+      "status": "pass",
+      "user": "missinglink",
+      "issue": "https://github.com/pelias/openstreetmap/issues/28",
+      "in": {
+        "sources": "osm",
+        "layers": "street",
+        "text": "rushendon furlong, england"
+      },
+      "expected": {
+        "coordinates": [ -0.636911, 51.83567 ],
```

Comment:
Cool thing about this street is that there are no housenumbers for it in OSM, so this will be the first time the residents of `Rushendon Furlong` can find their way home again.